reference/system-databases/inspection-summary.md

```diff
@@ -0,0 +1,75 @@
+category: reference
+---
+
+# INSPECTION_SUMMARY
+
+In some scenarios, you might pay attention only to the monitoring summary of specific links or modules. For example, the number of threads for Coprocessor in the thread pool is configured as 8. If the CPU usage of Coprocessor reaches 750%, you can determine that a risk exists that Coprocessor might become a bottleneck in advance. However, some monitoring metrics vary greatly due to different user workloads, so it is difficult to define certain thresholds. It is important to troubleshoot issues in this scenario, so TiDB provides the `inspection_summary` table for link summary.
+
+The structure of the `information_schema.inspection_summary` inspection summary table is as follows:
+
+{{< copyable "sql" >}}
+
+```sql
+desc inspection_summary;
+```
+
+```sql
++--------------+-----------------------+------+------+---------+-------+
+| Field        | Type                  | Null | Key  | Default | Extra |
++--------------+-----------------------+------+------+---------+-------+
+| RULE         | varchar(64)           | YES  |      | NULL    |       |
+| INSTANCE     | varchar(64)           | YES  |      | NULL    |       |
+| METRICS_NAME | varchar(64)           | YES  |      | NULL    |       |
+| LABEL        | varchar(64)           | YES  |      | NULL    |       |
+| QUANTILE     | double unsigned       | YES  |      | NULL    |       |
+| AVG_VALUE    | double(22,6) unsigned | YES  |      | NULL    |       |
+| MIN_VALUE    | double(22,6) unsigned | YES  |      | NULL    |       |
+| MAX_VALUE    | double(22,6) unsigned | YES  |      | NULL    |       |
++--------------+-----------------------+------+------+---------+-------+
+```
+
+Field description:
+
+* `RULE`: Summary rules. New rules are being added, and you can execute the `select * from inspection_rules where type='summary'` statement to check the latest rule list.
+* `INSTANCE`: The monitored instance.
+* `METRIC_NAME`: The monitoring metrics name.
+* `QUANTILE`: Takes effect on monitoring tables that contain `QUANTILE`. You can specify multiple percentiles by pushing down predicates. For example, you can execute `select * from inspection_summary where rule='ddl' and quantile in (0.80, 0.90, 0.99, 0.999)` to summarize the DDL-related monitoring metrics and query the P80/P90/P99/P999 results. `AVG_VALUE`, `MIN_VALUE`, and `MAX_VALUE` respectively indicate the average value, minimum value, and maximum value of the aggregation.
+
+> **Note:**
+>
+> Because summarizing all results causes overhead, the rules in `information_summary` are triggered passively, which means that the specified `rule` is displayed in the SQL predicate before the rule runs. For example, executing the `select * from inspection_summary` statement returns an empty result set. Executing `select * from inspection_summary where rule in ('read-link', 'ddl')` summarizes the read link and DDL-related monitoring metrics.
```

Comment:
```suggestion
> Because summarizing all results causes overhead, the rules in `information_summary` are triggered passively. That is, the specified `rule` runs only when it displays in the SQL predicate. For example, executing the `select * from inspection_summary` statement returns an empty result set. Executing `select * from inspection_summary where rule in ('read-link', 'ddl')` summarizes the read link and DDL-related monitoring metrics.
```